contributing-guides/git-terminal.md

```diff
@@ -34,19 +34,19 @@ Typically a pull request will include changes in a single file.
 
 # Updating your fork
 
-Forks aren't updated automatically and in order to avoid merge conflicts it should be done manually. This is possible either via the GitHub web interface by pressing `Fetch upstream` and `Fetch and merge` or using Git in the terminal:
+Forks aren't updated automatically and in order to avoid merge conflicts it should be done manually and regularly. This is possible via either the GitHub web interface by pressing `Fetch upstream` and `Fetch and merge` on the fork or using Git in the terminal:
 ```bash
 git checkout main
-git remote add upstream git@github.com:tldr-pages/tldr.git # only run this if you don't already have the upstream remote for https://github.com/tldr-pages/tldr.git
+git remote add upstream https://tldr-pages/tldr.git # only run this if you don't already have the upstream remote for https://github.com/tldr-pages/tldr.git
 git fetch upstream main
 git rebase upstream/main
 git push --force # not needed if you only want to update your local repository
 ```
 
 # Changing the email of your last commit
 
-If you used an email for a commit, which isn't added to your GitHub account, you can either add it [here](https://github.com/settings/emails) or change the email of your last commit with
+If the email that you used for a commit isn't added to your GitHub account, you can either add it [here](https://github.com/settings/emails) or change the email of your last commit with
 ```bash
-git commit --amend --author="Name <new@email.com>"
+git commit --amend --author="Name <new.email@example.com>"
```

Comment:
I think dots aren't allowed in all the email clients
```suggestion
git commit --amend --author="Name <new_email@example.com>"
```

Reply:
`new.email` is what @bl-ue suggested [here](https://github.com/tldr-pages/tldr/pull/5956#discussion_r631783099)

Reply:
Should I change it anyway?

Reply:
dots are surely allowed, infact it seems to be the acceptable way to separate words

Reply:
I agree with new.mail

Reply:
This should be resolved.